README.md

```diff
@@ -1,113 +1,93 @@
 ![Firebase Genkit logo](docs/resources/genkit-logo-dark.png#gh-dark-mode-only 'Firebase Genkit')
 ![Firebase Genkit logo](docs/resources/genkit-logo.png#gh-light-mode-only 'Firebase Genkit')
 
-Firebase Genkit (beta) is a framework with powerful tooling to help app developers build, test, deploy, and monitor AI-powered features with confidence. Genkit is also built from the ground up to be cloud optimized and code-centric, designed to feel familiar to most developers with less new concepts to learn. The Genkit framework itself is free and [open source](./LICENSE), and integrates with many services that have free tiers to get started.
+Firebase Genkit is a framework for TypeScript and Go, designed to help you build and iterate on AI-powered applications and features. It provides open source libraries and integrated tooling to support your AI app development.
 
-Genkit is available for [TypeScript (Node.js)](https://www.npmjs.com/package/genkit), with [Go support](https://github.com/firebase/genkit/tree/main/go) in active development.
+## Who is Genkit for?
 
-To be notified when Genkit for Go is ready for early community testing and get invited to our testers chat, [submit this form](https://docs.google.com/forms/d/e/1FAIpQLSeMYYFlH7bpsTZLWYDFSrgUG-64gkB3w_zXuF7NdTrZG04BLQ/viewform?resourcekey=0-eu_VfD8prb0igHZY5-oQKQ).
+Genkit is for any developer who wants to integrate generative AI capabilities into their apps using TypeScript or Go. Code written with Genkit libraries can be deployed anywhere that supports Node.js and Go runtimes, and the framework supports any model and vector store provider.
```

Comment:
if you say "the framework supports any model and vector store provider", you should somehow clarify that we have built-in support for certain providers and a plug-in system to support others